test-utils/runtime/src/genesismap.rs

```diff
@@ -81,12 +81,22 @@ impl GenesisConfig {
 		// Assimilate the system genesis config.
 		let mut storage =
 			Storage { top: map, children_default: self.extra_storage.children_default.clone() };
-		<system::GenesisConfig as GenesisBuild<Runtime>>::assimilate_storage(
-			&system::GenesisConfig { authorities: self.authorities.clone() },
+
+		<substrate_test_pallet::GenesisConfig as GenesisBuild<Runtime>>::assimilate_storage(
+			&substrate_test_pallet::GenesisConfig { authorities: self.authorities.clone() },
 			&mut storage,
 		)
 		.expect("Adding `system::GensisConfig` to the genesis");
 
+		<pallet_babe::GenesisConfig as GenesisBuild<Runtime>>::assimilate_storage(
```

Comment:
You already answer me somewhere else. I was asking to reserve the same treatment to AURA, GRANDPA, and the other stuff we're implementing the runtime Api for and for whom a backing proper pallet (instead of the "ad-hoc" runtime logic) could be beneficial